package.json

```
@@ -53,26 +51,15 @@
     "@types/yauzl-promise": "2.1.0",
     "codecov": "3.7.1",
     "cpy-cli": "2.0.0",
-    "jest": "27.3.1",
     "lint-staged": "9.2.5",
     "nyc": "14.1.1",
     "pre-commit": "1.2.2",
     "prettier": "1.15.3",
     "source-map-support": "0.5.10",
-    "ts-jest": "27.0.7",
-    "typescript": "4.9.3"
+    "typescript": "4.9.3",
+    "vitest": "^3.0.2"
```

```suggestion
    "vitest": "3.0.2"
```
Prefer pinning version.